source/_integrations/zha.markdown

```diff
@@ -419,6 +419,45 @@ After restoring a Home Assistant backup, you can re-configure ZHA and migrate to
 
+## Migrating to a new Zigbee coordinator adapter inside ZHA
+
+Follow this guide if you have a Zigbee Home Assistant (ZHA) network running and want to migrate from one Zigbee coordinator radio adapter to another Zigbee coordinator radio adapter.
+
+### Prerequisites
+
+- Your old Zigbee Coordinator radio adapter is used in the ZHA integration (not in deCONZ or MQTT)
+- It is of radio type ezsp (Silicon Labs EmberZnet), znp (Texas Instruments Z-Stack ZNP), or deCONZ (ConBee/RaspBee from dresden elektronik).
+  - If your old Zigbee coordinator is a deCONZ (ConBee/RaspBee) radio adapter, make sure it is running [firmware 0x26700700 (from 2021-08-18)](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Firmware-Changelog) or later.
+
+
+### To migrate to a new Zigbee coordinator radio inside ZHA
+
+1. Go to **{% my integrations title="Settings > Devices & Services" %}** and select the ZHA integration. Then select **Configure**.
+1. Under **Network settings**, select **Migrate radio**.
+1. Reconfiguration of ZHA will start. Select **Submit**.
+1. Under **Migrate or re-configure**, select **Migrate to a new radio**.
+1. **Migrate to a new radio** will inform you that an automatic backup will be performed and that your old Zigbee coordinator radio will then be reset before the backup is automatically restored.
+    - Select **Submit**.
+1. **Unplug your old radio** will inform you that your old Zigbee coordinator radio has been reset and that you can now plug in your new Zigbee coordinator radio adapter.
+    - To avoid interference, use a USB extension cable. 
+    - Use a USB 2.0 port or a powered USB 2.0 hub and keep the Zigbee stick away from USB 3.0 devices. 
+    - You may now also choose to either unplug your old Zigbee coordinator radio adapter or keep your old radio plugged in.
+    - If that radio was a combined Z-Wave and Zigbee radio, like the HUSBZB-1 adapter, then only the Zigbee radio part of it was reset. Confirm that the Zigbee Coordinator radio adapter is properly connected and select **Submit**.
+1.  You now need to start the backup restore process. 
+    - If your new Zigbee Coordinator radio adapter was automatically discovered by the Zigbee Home Automation (ZHA) integration, then select **Configure**.  
+    - If not, then you might first have to restart the radio migration flow. Under **Network Settings**, select **Migrate radio** > **Reconfigure ZHA** > **Migrate to a new radio**.
+1. Under **Network Formation**, select **Restore and automatic backup**.
+1. Under **Restore Automation Backup**, choose the latest automatic backup and select **Submit**.
+1. Under **Overwrite Radio IEEE Address**, select the **Permanently replace the radio IEEE address** and select **Submit**. Overwriting the IEEE address may take a while. Wait. 
```

Comment:
This screen won't appear for certain types of coordinators, or for new EZSP coordinators with firmware supporting the token interface:

```suggestion
1. If your radio requires overwriting the IEEE address, you will see a screen titled **Overwrite Radio IEEE Address**. Check the **Permanently replace the radio IEEE address** box and click **Submit**. Overwriting the IEEE address may take a while.
```